src/app/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import "./globals.css";
+import { Inter } from "next/font/google";
+import { ReactNode } from "react";
+import { PlayersProvider } from "@/context/playersContext";
+import { GameStateProvider } from "@/context/gameState.context";
+import { GameScoreProvider } from "@/context/gameScore.context";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata = {
+  title: "Tic Tac Toe",
+  description: "Tic Tac Toe Game",
+};
+
+export default function RootLayout({ children }: { children: ReactNode }) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <PlayersProvider>
+          <GameScoreProvider>
```

Comment:
I don't think we need `GameScoreProvider` to be wrapping the whole app, just the component where we actually right? Like the game selecion mode screen doesn't care about this. We should move this down